File: Xevenshtein.Droid/MainActivity.cs
using System;
using Android.App;
using Android.Content;
using Android.Runtime;
using Android.Views;
using Android.Widget;
using Android.OS;
using Xevenshtein.Core;

namespace Xevenshtein.Droid
{
    [Activity(Label = "Xevenshtein for Android",
        MainLauncher = true, Icon = "@drawable/icon")]
    public class MainActivity : Activity
    {
        private Button computeButton;
        private EditText firstWordTextInput, secondWordTextInput;
        private TextView resultTextView;
        protected override void OnCreate(Bundle bundle)
        {
            base.OnCreate(bundle);

            // Set our view from the "main" layout resource
            SetContentView(Resource.Layout.Main);
            computeButton = FindViewById<Button>(Resource.Id.ComputeButton);
            firstWordTextInput = FindViewById<EditText>(Resource.Id.FirstWordTextInput);
            secondWordTextInput = FindViewById<EditText>(Resource.Id.SecondWordTextInput);
            resultTextView = FindViewById<TextView>(Resource.Id.ResultTextView);

            computeButton.Click += (s, a) =>
            {
                var resultado = Levenshtein.Compute(firstWordTextInput.Text, secondWordTextInput.Text);
                resultTextView.Text = resultado.ToString();
            };
        }
    }
}

File: Android Studio Projects/AndroidXevenshtein/app/src/main/java/org/fferegrino/android/xevenshtein/MainActivity.java
package org.fferegrino.android.xevenshtein;

import android.app.Activity;
import android.support.v7.app.ActionBarActivity;
import android.os.Bundle;
import android.view.Menu;
import android.view.MenuItem;
import android.view.View;
import android.widget.*;

import org.fferegrino.android.xevenshtein.core.Levenshtein;


public class MainActivity extends ActionBarActivity {

    private Button computeButton;
    private EditText firstWordTextInput, secondWordTextInput;
    private TextView resultTextView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.activity_main);

        computeButton = (Button) findViewById(R.id.ComputeButton);
        firstWordTextInput = (EditText) findViewById(R.id.FirstWordTextInput);
        secondWordTextInput = (EditText) findViewById(R.id.SecondWordTextInput);
        resultTextView = (TextView) findViewById(R.id.ResultTextView);

        computeButton.setOnClickListener(new View.OnClickListener() {
            public void onClick(View v) {
                String result = new  Integer(Levenshtein.Compute(firstWordTextInput.getText().toString(), secondWordTextInput.getText().toString())).toString();
                resultTextView.setText(result.toCharArray(),0, result.length());
            }
        });
    }
}
